implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -399,22 +398,41 @@ impl NodeManager {
         Ok(())
     }
 
-    async fn initialize_defaults(&mut self, ctx: &Context) -> Result<()> {
+    async fn initialize_defaults(
+        &mut self,
+        ctx: &Context,
+        api_flow_control_id: &FlowControlId,
+    ) -> Result<()> {
         // Start services
+        ctx.flow_controls().add_consumer(
+            DefaultAddress::IDENTITY_SERVICE,
+            api_flow_control_id,
+            FlowControlPolicy::SpawnerAllowMultipleMessages,
+        );
         self.start_identity_service_impl(ctx, DefaultAddress::IDENTITY_SERVICE.into())
             .await?;
         self.start_authenticated_service_impl(ctx, DefaultAddress::AUTHENTICATED_SERVICE.into())
             .await?;
```

Comment:
Not adding these two services as consumers means that they are not accessible from the outside?

Reply:
I think IDENTITY_SERVICE is added above. Not sure about AUTHENTICATED_SERVICE, is it used? Should it be exposed via api transport?

Reply:
At least none of the tests/bats tests expect that